src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/3.diff.sql

```diff
@@ -0,0 +1,153 @@
+/*************************************************************
+    Instance Schema
+**************************************************************/
+CREATE TABLE dbo.InstanceSchema
+(
+    Name varchar(64) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    CurrentVersion int NOT NULL,
+    MaxVersion int NOT NULL,
+    MinVersion int NOT NULL,
+    Timeout datetime2(7) NOT NULL
```

Comment:
Do we need/want this much precision?

Reply:
Good point! Since for now considering 2 mins for timeout, we even dont need milliseconds. Fixed it to datetime2(0).